Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1744,6 +1744,29 @@ public RelativeDailyVolume RDV(Symbol symbol, int period = 2, Resolution resolut
             return R(symbol, mirrorOption, riskFreeRate, dividendYield, optionModel, ivModel, resolution);
         }
 
+
+        /// <summary>
+        /// Creates a new Stochastic RSI indicator which will compute the %K and %D
+        /// </summary>
+        /// <param name="symbol">The symbol whose Stochastic RSI we seek</param>
+        /// <param name="rsiPeriod">The period of the relative strength index</param>
+        /// <param name="stochPeriod">The period of the stochastic indicator</param>
+        /// <param name="kSmoothingPeriod">The smoothing period of K output</param>
+        /// <param name="dSmoothingPeriod">The smoothing period of D output</param>
+        /// <param name="movingAverageType">The type of moving average to be used</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to the Value property of BaseData (x => x.Value)</param>
+        /// <returns>A StochasticRelativeStrengthIndex configured with the specified periods and moving average type</returns>
+        [DocumentationAttribute(Indicators)]
+        public StochasticRelativeStrengthIndex SRSI(Symbol symbol, int rsiPeriod, int stochPeriod, int kSmoothingPeriod, int dSmoothingPeriod, MovingAverageType movingAverageType = MovingAverageType.Simple,
```

Comment:
Ah realizing it would be nice to add some nice/expected default values for these periods? What do other platforms normally use as defaults 👍

Reply:
talipp doesn't provide default values for indicators but this is something which is considered
`https://github.com/nardew/talipp/issues/98`
Tulip doesn't provide default values also https://github.com/TulipCharts/tulipindicators/blob/master/indicators/stochrsi.c
Same for Talib 
https://github.com/TA-Lib/ta-lib/blob/main/src/ta_func/ta_STOCHRSI.c

I don't think others all Lean indicators have default parameters (but I may be wrong as I don't know all Lean indicators code). On the other side I noticed that STC have default values.

Reply:
What is odd is that default values seems to be defined in indicator cs file not in QCAlgorithm.Indicators.cs